src/doby-grid.js

```diff
@@ -3675,6 +3675,24 @@ var DobyGrid = function (options) {
 		if (allowed.indexOf(format) < 0) throw new Error('Sorry, "' + format + '" is not a supported format for export.');
 		callback = callback || function () {};
 
+		var htmlTagToEscape = {
```

Comment:
Hey Chedwick,

I think the following code will ensure more cases are covered:

 var entityMap = {
    "&": "&amp;",
    "<": "&lt;",
    ">": "&gt;",
    '"': '&quot;',
    "'": '&#39;',
    "/": '&#x2F;'
  };

  function escapeHtml(string) {
    return String(string).replace(/[&<>"'\/]/g, function (s) {
      return entityMap[s];
    });
  }
Can you also please include a unit test to catch this bug?